src/App.jsx

```diff
@@ -1,44 +1,28 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { tabs } from './constants/constants';
+import { Tabs } from './components/Tabs';
+import { getActiveTabTitle } from './helpers/getActiveTabTitle';
 
-export const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-];
+const startRendTabsId = tabs[0].id;
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [tabActiveId, setTabActiveId] = useState(startRendTabsId);
```

Comment:
you can store whole tab in state:
```suggestion
  const [activeTab, setActiveTab] = useState(tabs[0]);
```